### PR TITLE
02-keras.md:; simplify code lines

### PR DESCRIPTION
**Pull request by wmotion**
The variables names are lower-case. 
All penguin-related variable are named `penguins_*`. 
An output line has been changed since I obtained a slightly different output.

### CODE REVIEW

**Comment by wmotion** [comment]
Please note that in [Section 7 on performing a prediction](https://carpentries-incubator.github.io/deep-learning-intro/02-keras/index.html#7-perform-a-predictionclassification) I obtained different results. For example, the output of `prediction` is 
```
      Adelie     Chinstrap    Gentoo
0   0.999959  3.145577e-08  0.000041
1   0.983226  2.183303e-03  0.014590
2   0.999926  1.490188e-06  0.000072
3   0.035932  2.511340e-04  0.963817
4   0.953162  2.092293e-02  0.025915
..       ...           ...       ...
64  0.001576  9.657394e-01  0.032685
65  0.043842  7.111872e-01  0.244971
66  0.153382  7.816030e-01  0.065015
67  0.491400  3.553052e-05  0.508564
68  0.959014  2.281321e-08  0.040986

[69 rows x 3 columns]
```

As a result, the confusion matrix and its visualisation in [Section 8 on measuring performance](https://carpentries-incubator.github.io/deep-learning-intro/02-keras/index.html#8-measuring-performance) are quantitatively and qualitatively different. 
**If** this can be reproduced, the comments in the [box](https://carpentries-incubator.github.io/deep-learning-intro/02-keras/index.html#confusion-matrix-1) should be edited accordingly.

Please double check this, besides the commit.

**Comment by wmotion** [comment]
The table [above](https://github.com/carpentries-incubator/deep-learning-intro/pull/181#issuecomment-1020071947) showed up also during the workshop at the [e-ScienceCenter in the Netherlands](https://esciencecenter-digital-skills.github.io/2022-01-25-ds-dl-intro/). This strengthens the case for a revision of the workshop material.

**Comment by psteinb** [comment]
As a general comment, naming `X` capitalized is a standard in the `sklearn` community. I think we should keep it like this at it aligns to the `sklearn` docs well.

**Comment by psteinb** [comment]
For the reproducibility issue, I suggest to disentangle this from this PR and open an issue.

File: _episodes/02-keras.md [hunk 1]
```diff
@@ -432,7 +432,7 @@ The model summary here can show you some information about the neural network we
 > > {:.language-python}
 > >
 > > ~~~
-> Model: "functional_1"
+> Model: "model_1"
```
Comment (dafnevk) [inline _episodes/02-keras.md:435]: this one probably depends on tensorflow versions, in my environment (tensorflow=2.4.1) I get the name `functional_3`. Not too important I guess, should we make a note on it? We could also define the name manually.

File: _episodes/02-keras.md [hunk 2]
```diff
@@ -147,7 +147,7 @@ This will give you a pandas dataframe which contains the penguin data.
 > > ~~~
 > > {:.language-python}
 > > ~~~
-> > ['Adelie', 'Chinstrap', 'Gentoo']
+> > array(['Adelie', 'Chinstrap', 'Gentoo'], dtype=object)
```
Comment (dafnevk) [inline _episodes/02-keras.md:150]: I indeed also get this output.

**Review by dafnevk — CHANGES_REQUESTED** [review]
Changing the output lines is fine with me.
I would leave the variable names as they are with capitalized X, as is the convention with sklearn.

File: _episodes/02-keras.md (hunk 1 above)
Comment (psteinb) [reply _episodes/02-keras.md:435]: A note or call-out box makes sense from my point of view. Especially considering self-paced learners. @wmotion could you include that?

Comment (wmotion) [reply _episodes/02-keras.md]: I have added one line mentioning that name differences are not important.
https://github.com/wmotion/deep-learning-intro/blob/4057dc0b9294a13ebe73022618d7ca11de3c5cd2/_episodes/02-keras.md?plain=1#L454

**Review by dafnevk — APPROVED** [review]

**Pull request merged**